cob/cob.c

```diff
@@ -335,6 +335,7 @@ static int ns_cmp(const void *key0, const void *key1)
 				(const struct m0_cob_nskey *)key1);
 }
 
+#ifdef NEW_BTREE_INTEGRATION_COMPLETE
```

Comment:
I forgot to remove the structure cob_ns_ops below. I will take care of them. Please remove the #ifdef NEW_BTREE_INTEGRATION_COMPLETE.

Reply:
Done.